### ISSUE TITLE
Cloning:Warning: You appear to have cloned an empty repository.

### ISSUE THREAD

**Issue by mariaclaudia** [neevany/caravel-on-heroku#6]
As suggested, in order to create the user, I've used the command:
`heroku git:clone --app appName`
but the message I received is:
`warning: You appear to have cloned an empty repository.`
The folder is created but it's empty, the repo is not cloned locally

**Comment by neevany** [comment]
First you need to deploy it in heroku using the deploy to heroku button. then follow the steps.

[<a href="https://heroku.com/deploy?template=https://github.com/neevany/caravel-on-heroku"><img src="https://camo.githubusercontent.com/83b0e95b38892b49184e07ad572c94c8038323fb/68747470733a2f2f7777772e6865726f6b7563646e2e636f6d2f6465706c6f792f627574746f6e2e737667" alt="Deploy" data-canonical-src="https://www.herokucdn.com/deploy/button.svg" style="max-width:100%;"></a>](url)

**Comment by mariaclaudia** [comment]
Yes , I did it. Here a workaround from Heroku: https://kb.heroku.com/why-do-i-see-a-message-you-appear-to-have-cloned-an-empty-repository-when-using-heroku-git-clone

**Comment by YJF713** [comment]
这是一个博客

**Comment by YJF713** [comment]
这是一个博客
